packages/quantic/playwright/utils/requests.ts

```diff
@@ -57,3 +57,11 @@ export function isCollectEvent(request: Request): boolean {
     request.method() === 'POST' && analyticsCollectUrlRegex.test(request.url())
   );
 }
+
+/**
+ * Indicates whether the specified request corresponds to a search api request.
+ * @param request The request to check.
+ */
+export function isSearchRequest(request: Request): boolean {
```

Comment:
surprised we didnt have something for that already :O

Reply:
same lol